lib/controller.js

```diff
@@ -941,6 +943,8 @@ controller.updateTesselWithVersion = function(opts, tessel, currentVersion, buil
 controller.tesselEnvVersions = opts => {
   opts.authorized = true;
   return controller.standardTesselCommand(opts, tessel => {
+    var pkg = path.join(__dirname, '..', 'package.json');
+    var cliVersion = fs.readJsonSync(pkg).version;
```

Comment:
Private conversation in Slack with a solution:

```
rwaldron [7:26] 
Since your plan is to just include that value in the display, we don't really need to care about ​_what_​ the value is, just that it "is"

[7:30] 
So! You can get the ​_expected_​ value from the package.json in your test! Does that make sense?

wyze [7:31 AM] 
yeah just require in the package.json in the test? I was going to do that too, but it almost felt a little dirty. lol

rwaldron [7:31 AM] 
That means you can get rid of the stub all together, let it do the full operation, because it relies on invariants

[7:31] 
Naw, it's cool
```